README.md

```diff
@@ -8,152 +8,225 @@
 
 [Toolbox](https://github.com/containers/toolbox) is a tool that offers a
 familiar package based environment for developing and debugging software that
-runs fully unprivileged using [Podman](https://podman.io/).
+runs fully rootless using [Podman](https://podman.io/) and integrates with
+your system.
 
-The toolbox container is a fully *mutable* container; when you see
-`yum install ansible` for example, that's something you can do inside your
-toolbox container, without affecting the base operating system.
-
-This is particularly useful on
-[OSTree](https://ostree.readthedocs.io/en/latest/) based operating systems like
-[Fedora CoreOS](https://coreos.fedoraproject.org/) and
-[Silverblue](https://silverblue.fedoraproject.org/).  The intention of these
-systems is to discourage installation of software on the host, and instead
-install software as (or in) containers.
-
-However, this tool doesn't *require* using an OSTree based system — it
-works equally well if you're running e.g. existing Fedora Workstation or
-Server, and that's a useful way to incrementally adopt containerization.
-
-The toolbox environment is based on an [OCI](https://www.opencontainers.org/)
-image. On Fedora this is the `fedora-toolbox` image. This image is used to
-create a toolbox container that seamlessly integrates with the rest of the
-operating system.
-
-## Usage
+## Basic usage
 
 ### Create your toolbox container:
-```
+
+```shell
 [user@hostname ~]$ toolbox create
-Created container: fedora-toolbox-30
+Created container: fedora-toolbox-32
 Enter with: toolbox enter
 [user@hostname ~]$
 ```
-This will create a container called `fedora-toolbox-<version-id>`.
+
+This will create a container called `fedora-toolbox-<version-id>` based on the
+version of Fedora you are using.
 
 ### Enter the toolbox:
-```
+
+```shell
 [user@hostname ~]$ toolbox enter
 ⬢[user@toolbox ~]$
 ```
 
-## Dependencies and Installation
+> Notice: The '⬢' symbol may not be present if you are using a custom shell
+> theme.
+
+### Remove a toolbox container:
+
+```shell
+[user@hostname ~]$ toolbox rm <container-name>
+[user@hostname ~]$
+```
+
+## Tell me more about Toolbox!
```

Comment:
I wonder if we can compress this section a bit.

Currently, on this ThinkPad x220, I have to press *page down* ten times to scroll through the entire `README.md`. That's a lot of text! It might be better to aim to have each section fit on one page, with the exception of tables and lists which tend to consume more space.

Since the introductory sentence says *Toolbox ... is a ... environment*, maybe we can start this section more directly? eg., *The Toolbox environment is a Podman container ..." or something on those lines. We don't need to explain too much about what's OSTree or CoreOS or Silverblue. It's enough to link to some nice references and let the reader figure out all the good things like fearless and robust upgrades.